src/Core/CommandLine.cs

```diff
@@ -386,6 +404,13 @@ private static void SetConfig(Config config)
                 }
             }
 
+            // set burst headers service name
+            if (string.IsNullOrWhiteSpace(Config.BurstService))
+            {
+                VersionExtension.GetReflectionValues();
```

Comment:
Does GetReflectionValues return values?  If not, I would use Init or Load or something indicating what it does.

It might be cleaner to have .Name call that under the covers.